### DISCUSSION TITLE
Ошибка при скачивании страницы: server returned 300 Multiple Choices

### DISCUSSION THREAD

**Discussion by vladislavgvozdik** [lirix360/ReadmangaGrabber#72] [Q&A] [answered]
Раньше скачивал все прекрасно работало, сегодня решил скачать новую мангу, вылазит такая ошибка и в папке пустой пдф. Сначала подумал, что дело в самой манге, но при попытке скачать предыдущую такая же ситуация

**Comment by vladislavgvozdik** [comment]
Скачиваю с readmanga

**Reply by Meloni4** [reply]
У меня такая же ошибка и как её исправить непонятно

**Reply by Meloni4** [reply]
Хотя с Mangalib всё скачивается как нужно

**Comment by lirix360** [comment, marked as answer]
@vladislavgvozdik @Meloni4 Исправлено в версии 2.1.4

**Reply by Meloni4** [reply]
Thanks)
